### PR TITLE
case 21395: Adjust culling matrix 

### PR DESCRIPTION
**Pull request by amerhifi**
https://highfidelity.manuscript.com/f/cases/21395/Adjust-culling-matrix-to-fix-vertical-culling

Adjust culling matrix so that items are not cut off from rendering at various pitch angles.  

Test:

Log into quest-demo domain
Go to the board room
Stand at end of table
Look up as far as you can while your eyes look at the table and chairs. 
The table should stay in view at all times, as well as the chairs
Look down, and repeat with the ceiling and lights. 

It should appear as if nothing is being dropped and recreated in the view based on the angle of your head.

### CODE REVIEW

File: libraries/oculusMobilePlugin/src/OculusMobileDisplayPlugin.cpp [hunk 1]
```diff
@@ -136,9 +137,12 @@ glm::mat4 OculusMobileDisplayPlugin::getCullingProjection(const glm::mat4& baseP
         for (size_t i = 0; i < 2; ++i) {
             fovs[i].extract(trackingState.Eye[i].ProjectionMatrix);
         }
+
         fovs[0].extend(fovs[1]);
-        return fovs[0].withZ(baseProjection);
+        result= glm::scale( fovs[0].withZ(baseProjection),glm::vec3(1.5f));
```
Comment (amerhifi) [inline libraries/oculusMobilePlugin/src/OculusMobileDisplayPlugin.cpp]: Please keep me honest here and if this is not the preferred way, I would love to hear suggestions for better way of going about this.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17544/HighFidelity-Beta-PR14999-5c1e5e0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17544/HighFidelity-Beta-Interface-PR14999-5c1e5e0.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17544/HighFidelity-Beta-PR14999-5c1e5e0.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17544/HighFidelity-Beta-PR14999-5c1e5e0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17544/HighFidelity-Beta-Interface-PR14999-5c1e5e0.exe

**Pull request merged**